package.json

```diff
@@ -34,6 +34,7 @@
     "deep-object-diff": "1.1.0",
     "env-cmd": "10.1.0",
     "graphql": "15.3.0",
+    "graphql-tag": "^2.11.0",
```

Comment:
hm?

Reply:
🤷  should this not be here? Is there something else I should do? I just did `npm install`

Reply:
Before it wasn't compiling because of something to do with graphql-tag but 🤷  seems fine now so I'll delete it.